Drivers/Inc/Contactors.h

```diff
@@ -7,39 +7,35 @@
 #include "config.h"
 #include "BSP_GPIO.h"
 
-#define CONTACTORS_PORT PORTC
-#define MOTOR_CNCTR_PIN 2
-#define ARRAY_CNCTR_PIN 1 
 
-typedef enum {MOTOR=0, ARRAY, NUM_CONTACTORS} contactor_t;
+typedef enum {
+    ARRAY_CONTACTOR = 0,
+    ARRAY_PRECHARGE,
+    MOTOR_CONTACTOR,
+    NUM_CONTACTORS
+} contactor_t;
 
 /**
  * @brief   Initializes contactors to be used
  *          in connection with the Motor and Array
- * @param   contactor the contactor to initialize
  * @return  None
- */ 
-void Contactors_Init(contactor_t contactor);
+ */  
+void Contactors_Init();
```

Comment:
Is this a breaking change anywhere that you know of?

Reply:
Only in the tests